tox.ini

```diff
@@ -56,7 +56,7 @@ commands =
 
 [testenv:mypy]
 commands =
-    mypy {[vars]PACKAGE_MODULE} {[vars]PACKAGE_TESTS}
+    mypy {[vars]PACKAGE_MODULE}
```

Comment:
I don't think we really want to deal with `mypy` in the test directory. I can revert this if you disagree (I will have to add approximately 20 comments `#type: ignore...` though).

Reply:
I think using mypy still make sense because it checks if a data you're giving to tested objects are indeed supported. What are the reasons for the ignoring types?

Reply:
Ok, I have resolved all of the issues. The type: ignore was needed for fixtures which read JSONs of an expected structure which I have typed with the expected return value. Actually the number 20 may have been slightly exaggerated :sweat_smile: